criteria/make-contributing-easy.md

```diff
@@ -43,5 +43,7 @@ order: 5
 
 ## Further reading
 
+* [GitHub Learning Lab](https://lab.github.com/) for learning git or refresh your skills.
+* [Git Cheat Sheet](https://education.github.com/git-cheat-sheet-education.pdf) a list with the most common used git commands.
```

Comment:
While good resources in general, these would make more sense in Maintain version control than Make contributing easy, and even there we try to be tool agnostic and not prescriptive about Git or GitHub.

Reply:
In order to ease the path and make it more accessible for everyone to work in the open, are we recommending anything or giving any tips for people that has never worked in an open source environment?

Reply:
We could. But if we were to add something, I think it should be about version control in general so that what they learn are applicable independently of the version control system or repository host and not favour one over the others.

Reply:
Feel free to add your favourite resources!

Reply:
How about adding this instead, which is more on the topic of making contributing easy, and is something the maintainer of repository can put into use rather than aimed at a new contributor?
[How to inspire exceptional contributions to your open-source project](https://www.netdata.cloud/blog/open-source-contributions/)

Reply:
> How about adding this instead, which is more on the topic of making contributing easy, and is something the maintainer of repository can put into use rather than aimed at a new contributor?
> [How to inspire exceptional contributions to your open-source project](https://www.netdata.cloud/blog/open-source-contributions/)

Okay, so you're observing that "learning git" links are less fitting for "make contributing easy" and that something along the lines of "inspire exceptional contributions" is more fitting. Now that I think about it, I agree. Would  "Maintain version control" be a better place for the learning git links? Perhaps with a note in for devs-and-designers to become familiar with the industry standard tools?